src/anbox/cmds/container_manager.cpp

```diff
@@ -63,6 +66,14 @@ anbox::cmds::ContainerManager::ContainerManager()
         WARNING("");
       }
 
+      if (!lxc_conf_path_.empty()) {
+        SystemConfiguration::instance().set_lxc_conf_path(lxc_conf_path_);
```

Comment:
I don't like this very much. Anbox is meant to abstract the container engine from the user. Providing a path to an existing LXC configuration file is something only useful for a developer. If we want additional features like memory or CPU limits we should implement this in a generic way so the LXC backend can write the right bits into the configuration file.

What we can start with is adding static CPU/memory limits and then add a configuration file for Anbox which allows users to change the limits if they want. Something like `anbox config set limits.cpus 4` or `anbox config set limits.memory 4G`

What do you think?

Reply:
yeah, you are right. I had considered your proposal as well before coding. but... i am just too lazy to write all of them...